### PR TITLE
Enable mocha api/test in circle.yml (and skip broken tests)

### PR DESCRIPTION
**Pull request by michielbdejong**
Continuation of #207

### CODE REVIEW

**Comment by michielbdejong** [comment]
Circleci is passing now \0/
Squashed and rebased.

File: api/test/utils.test.js [hunk 1]
```diff
@@ -15,6 +15,18 @@ describe('Utils', () => {
...
+    assert.isTrue(this.utils.isAccountUri('https://example'))
+    assert.isFalse(this.utils.isAccountUri('ftp://example'))
+  })
+
+  it.skip('detects foreign account URI', function () {
+    assert.isTrue(this.utils.isForeignAccountUri('http://example'))
+    assert.isFalse(this.utils.isForeignAccountUri(this.utils.ledgerUriPublic))
+    assert.isFalse(this.utils.isAccountUri('ftp://example'))
+  })
+
```
Comment (michielbdejong) [inline api/test/utils.test.js]: Oops, this is a merge mistake, will remove.

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "dev-with-proxy": "echo 'need permission to bind to ports 80 and 443' ; sudo echo -n ; DEBUG_COLORS=1 concurrently -p \"[{name}]\" -n \"watcher,ledger,api,wallet,proxy\" -c \"yellow,red,blue,cyan,green\" \"npm run watch-client\" \"npm run start-dev-ledger\" \"npm run start-dev-api\" \"npm run start-dev\" \"sudo npm run proxy\"",
     "test": "touch src/theme/variables-custom.scss ; karma start",
     "test-api": "better-npm-run test-api",
+    "test-mocha": "API_CONFIG_FILE=./api/test/data/env.list ./node_modules/.bin/mocha api/test",
```
Comment (dappelt) [inline package.json]: Is the config file (`./api/test/data/env.list`) actually used somewhere in the tests?

Comment (michielbdejong) [inline package.json]: Ah, just found https://github.com/interledgerjs/ilp-kit/blob/master/bin/normalizeEnv.js#L24 - will use that instead

**Comment by michielbdejong** [comment]
@dappelt [These changes](https://github.com/interledgerjs/ilp-kit/pull/216/files#diff-a354c9ed09a453efacfe601ecd033bf1) are probably going to break your integration tests, right? Is there a diagram of which component connects to which component on localhost in production, and in test environment? If not, can we maybe draw that on the whiteboard together so I get a better understanding of our test setup? Thanks! :)

**Comment by michielbdejong** [comment]
Thanks!!

**Pull request merged**